conf/messages

```diff
@@ -274,6 +274,24 @@ gains.sectionCompletedState.title = Have you finished this section?
 gains.sectionCompletedState.hint = You’ll still be able to go back and review the information that you’ve given us.
 gains.sectionCompletedState.error.required = Select if you’ve completed this section
 
+#################       Business Reliefs      ########################################
+
+business-reliefs.non-deductible.title = Non-deductible loan interest from investments into property letting partnerships
+business-reliefs.non-deductible.paragraph1.agent = Your client can claim tax relief on the cost of getting a loan to fund a partnership, or the interest on such a loan. Relief can also be claimed on the cost of getting alternative finance arrangements, and the interest on payments for such arrangements. This relief is limited to the extent that those funds are used for any part of the business that consists of residential properties.
+business-reliefs.non-deductible.paragraph2.agent = The tax relief your client can claim is restricted to the basic rate of Income Tax.
+business-reliefs.non-deductible.paragraph1.individual = You can claim tax relief on the cost of getting a loan to fund a partnership, or the interest on such a loan. Relief can also be claimed on the cost of getting alternative finance arrangements, and the interest on payments for such arrangements. This relief is limited to the extent that those funds are used for any part of the business that consists of residential properties.
+business-reliefs.non-deductible.paragraph2.individual = The tax relief you can claim is restricted to the basic rate of Income Tax.
+business-reliefs.non-deductible.paragraph3.1 = For more information, visit
+business-reliefs.non-deductible.paragraph3.link = Helpsheet 340
+business-reliefs.non-deductible.paragraph3.3 = and read the section titled
+business-reliefs.non-deductible.paragraph3.4 = Residential property finance cost restriction.
+business-reliefs.non-deductible.question.input = How much is the non-deductible loan interest from interest investments into property letting partnerships?
+business-reliefs.non-deductible.question.input.error.empty_field = Enter the value of the non-deductible loan interest
+business-reliefs.non-deductible.question.input.error.incorrect-characters = Non-deductible loan interest must only include the numbers 0-9 and a decimal point
+business-reliefs.non-deductible.question.input.error.max-amount.agent = "The amount of your non-deductible loan interest must be less than £100,000,000,000"
+business-reliefs.non-deductible.question.input.error.max-amount.individual = "The amount of your clients non-deductible loan interest must be less than £100,000,000,000"
```

Comment:
You've accidentally got quotes in the error message:
![Screenshot 2025-04-07 at 11 55 35](https://github.com/user-attachments/assets/ad17af04-9440-448b-9c5e-a3426294a32e)

Reply:
Removed quotes:

![Warning_Box](https://github.com/user-attachments/assets/e357358e-7a48-4157-ba0b-4deda2b8f7e5)
![Warning_Box_Welsh](https://github.com/user-attachments/assets/d2cc340f-d787-4b1a-a591-d70d8488732b)